docs/application/vstools-mac/overview.md

```diff
@@ -1,150 +1,137 @@
+
+1. In the Visual Studio Mac IDE menu, select **Visual Studio > Extensions**.
+
+   ![Install extension](media/install-extension1.png)
+
+2. In the **Extension Manager** dialog that appears, click **Gallery**.
+
+	![Extension Manager](media/install-extension2.png)
+   
+3. To get the latest extensions, click **Refresh**.
+
+	![Browse Extension](media/install-extension3.png)
+	
+4. After the extension list is refreshed, expand **IDE extensions**, select **Visual Studio for Mac Extension for Tizen**, and click **Install**.
+
+	![Install Popup](media/install-extension4.png)
+	![Install Popup1](media/install-extension5.png)
+
+5. In the **VisualStudio** dialog that appears, click **Install**.
+
+    ![Install Popup2](media/install-extension6.png) 
+
+   The extension is installed. 
+   
+   > **Note**
+   > 
+   > To complete the installation, ensure that you restart the IDE.
+   
+   After successful installation, **Tizen** appears in the project wizard.
+   
+   ![Project Wizard](media/install-extension7.png)
+
+## Install Tizen Baseline SDK
+
+Baseline SDK contains Tizen-specific libraries and tools such as profiler, debugger, and so on.
+
+After installing the Visual Studio Mac Extension for Tizen, you must set up the Tizen baseline SDK. You can either:
+
+- [Install a new Tizen baseline SDK](#install-new-tizen-baseline-sdk)
+- [Configure an existing Tizen baseline SDK](#configure-existing-tizen-baseline-sdk)
+
+### Install New Tizen Baseline SDK
+
+1. In the Visual Studio Mac IDE menu, go to **Tools > Tizen > Tizen Package Manager**.
+2. Select **Install new Tizen SDK**.
+  
+   ![Select new installation](media/howtoinstall-installwizard1.png)
+
+3. Read the license document and click **I Agree**.
+
+   ![Agree to license details](media/howtoinstall-installwizard2.png)
+
+4. Enter the root directory path where you want to install the Tizen baseline SDK and click **Next**.
+
+   ![Set the installation path](media/howtoinstall-installwizard3.png)
+
+   The Tizen SDK installer is downloaded and the baseline SDK is installed automatically.
+
+   ![Installer download](media/howtoinstall-installwizard4.png)
+
+5. In the **Package Manager** window that appears, select **Main SDK**.
+
+6. Select **Tizen SDK tools** and click **install**. You can see the installation progress in the **Progress** tab.
+
+   ![Tool installation](media/howtoinstall-installwizard6.png)
+
+<a name="configure-existing-tizen-baseline-sdk"></a>
+### Configure Existing Tizen Baseline SDK 
+
+You can also use Tizen Package Manager to configure the Tizen baseline SDK path and each tool path directly:
+
+- To set up the Tizen baseline SDK path:
+
+  1. In the Visual Studio Mac IDE menu, select **Tools > Tizen > Tizen Package Manager**.
+  2. Select **Use installed Tizen SDK**.
+
+     ![Baseline SDK Install](media/howtoinstall-installwizard7.png)
+
+  3. Enter the root directory of your existing Tizen Studio installation and click **OK**.

```

Comment:
I know Tizen SDK is installed by Tizen Studio. However, we are using the term, 'Tizen SDK' or 'Tizen baseline SDK' here instead of Tizen Studio. 
How about using the consistent term?

Reply:
Yes, changing to Done. Tizen Baseline SDK